server/CHANGELOG

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.3.0] ## 2022-03-18
+### Added  
```

Comment:
The addition/changes are missing.

Reply:
In order to win some times, I did the change myself. We can speak directly about what is coming next on monday.